Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:21.7.2-bookworm-slim as frontend-builder
+
+WORKDIR /app
+
+COPY ./frontend/package.json frontend/package-lock.json ./
+RUN npm install
+
+COPY ./frontend ./
+RUN npm run build
+
+
+FROM python:3.12-slim as runtime
+
+WORKDIR /app
+ENV PYTHONPATH '/app'
+
+RUN apt-get update -y
+RUN apt-get install -y curl make git python3.11 python3-pip
```

Comment:
is there a reason to install 3.11 in a 3.12 image?
the runtime image should also already have pip.

Reply:
Whoops good catch, copy/pasta